packages/fast-components-class-name-contracts-base/src/dialog/index.ts

```diff
@@ -2,7 +2,18 @@
  * The class name contract for the dialog component
  */
 export interface IDialogClassNameContract {
+    /**
+     * The root dialog component
+     */
     dialog: string;
+
+    /**
+     * The dialog backdrop
+     */
     dialog_modalOverlay: string;
+
+    /**
+     * The foregournd dialog content region
```

Comment:
I don't think "foreground" is relevant here - "The dialog content region" tells me everything I need to know

Reply:
Correct; there's not a background content region on the component so this would confuse me